src/App/Init.php

```diff
@@ -220,6 +220,7 @@
 	'config'               => $di->lazyNew(Ushahidi\App\Formatter\Config::class),
 	'dataproviders'        => $di->lazyNew(Ushahidi\App\Formatter\Dataprovider::class),
 	'country_codes'        => $di->lazyNew(Ushahidi\App\Formatter\CountryCode::class),
+	'hxl_licenses'        => $di->lazyNew(Ushahidi\App\Formatter\HXLLicense::class),
```

Comment:
Could we group all the HXL related DI code in 1 block? I think long term we should move DI into domain related groups (ie. Post, Tag, Form, User etc) rather than grouping by classes type

Reply:
@willdoran Just cc'ing you to check you agree?

Reply:
Yep, totally, sub-division-for-ease-of-editing-for-the-win

Reply:
yup

Reply:
if we pair today can we also look at how App/Init is different from src/Init ? It's kind of confusing . I moved the hxl code to App\Init for now, we can discuss it tonight